src/main/java/by/exadel/internship/service/impl/EmailServiceImpl.java

```diff
@@ -0,0 +1,55 @@
+package by.exadel.internship.service.impl;
+
+import by.exadel.internship.dto.formDTO.FormRegisterDTO;
+import by.exadel.internship.mail.EmailProperties;
+import by.exadel.internship.mail.EmailTemplate;
+import by.exadel.internship.service.EmailService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSenderImpl;
+import org.springframework.stereotype.Service;
+
+import javax.annotation.PostConstruct;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class EmailServiceImpl implements EmailService {
+
+    private final EmailProperties emailProperties;
+    private JavaMailSenderImpl mailSender;
+
+    private static final Map<String,String> MAIL_PROPS = Map.of("mail.transport.protocol", "smtp",
+                                                                "mail.smtp.auth", "true",
+                                                                "mail.smtp.starttls.enable", "true",
+                                                                "mail.debug", "true");
+
+    @PostConstruct
+    public void init() {
+        mailSender = new JavaMailSenderImpl();
+
+        mailSender.setHost(emailProperties.getHost());
+        mailSender.setPort(emailProperties.getPort());
+        mailSender.setUsername(emailProperties.getUsername());
+        mailSender.setPassword(emailProperties.getPassword());
+
+        Properties props = mailSender.getJavaMailProperties();
+        props.putAll(MAIL_PROPS);
+    }
+
+    public boolean sendSimpleMessage(FormRegisterDTO formRegisterDTO) {
+        EmailTemplate emailTemplate = new EmailTemplate();
+        SimpleMailMessage message = new SimpleMailMessage();
+        message.setFrom(emailTemplate.getFrom());
+        message.setTo(formRegisterDTO.getEmail());
+        message.setSubject(emailTemplate.getSubject());
+        message.setText(emailTemplate.getText());
+
+        mailSender.send(message);
```

Comment:
On exception it should return false and not exception

Reply:
done